lib/db/entity/sub_category.dart

```diff
@@ -0,0 +1,40 @@
+class SubCategory{
```

Comment:
table name should map properly to class name.
If the table name is sub_category, then the Dart name should be SubCategory
If the table name is subcategory, then the Dart name should be Subcategory